build_runner/lib/src/generate/directory_watcher_factory.dart

```diff
@@ -3,6 +3,8 @@
 // BSD-style license that can be found in the LICENSE file.
 import 'package:watcher/watcher.dart';
 
+export 'package:watcher/watcher.dart' show DirectoryWatcher;
```

Comment:
Fair enough. Was hoping to be able to keep some references in a Doc comment elsewhere, but I'll change them to backticks instead.